### PR TITLE
Scripts Form Config is not showing button icons

### PR DESCRIPTION
**Pull request by fdelcampo**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,7 @@ Version History
 v5.25.4
 -------
 
+* Scripts Form Config is not showing button icons `https://github.com/lsst-ts/LOVE-frontend/pull/541`_
```
Comment (sebastian-aranda) [inline CHANGELOG.rst:8]: You are missing a `>` after the `541`. Please always check the compiled version of the CHANGELOG to ensure things are properly displayed.

File: love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.jsx [hunk 2]
```diff
@@ -24,11 +24,14 @@ import { Rnd } from 'react-rnd';
 import yaml from 'js-yaml';
 import Form from '@rjsf/core';
 import rjsfValidator from '@rjsf/validator-ajv8';
+import { getSubmitButtonOptions } from '@rjsf/utils';
+// import { FormattedMessage } from 'react-intl';
```
Comment (sebastian-aranda) [inline love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.jsx:28]: This line should be removed.

File: love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.jsx [hunk 3]
```diff
@@ -810,23 +842,88 @@ export default class ConfigPanel extends Component {
...
+                  }
+                  .array-item {
+                    position: relative;
+                    display: block;
+                    padding: 0.5rem 1rem;
+                    text-decoration: none;
+                    border: 1px solid rgba(0,0,0,.125);
+                  }
+                  `}
+                  </style>
```
Comment (sebastian-aranda) [inline love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.jsx:908]: **PLEASE** move this out of here. Could you put them on the `module.css`?

**Review by sebastian-aranda — CHANGES_REQUESTED** [review]
Looks good, although I left a few comments, please take a look to them @fdelcampo.

File: love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.jsx (hunk 3 above)
Comment (fdelcampo) [reply love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.jsx:908]: This style is in this place for the html generated for the rjsf library. The class property in this html not can change for a className of the module.css

Comment (sebastian-aranda) [reply love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.jsx:906]: I think you can using the `:global` directive: https://stackoverflow.com/questions/39805537/how-to-apply-global-styles-with-css-modules-in-a-react-app, right?

Comment (fdelcampo) [reply love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.jsx:906]: Thanks

File: love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.module.css [hunk 4]
```diff
@@ -446,3 +446,63 @@ fieldset p {
...
+  padding-left: 0;
+  margin-bottom: 0;
+}
+:global(.array-item) {
+  position: relative;
+  display: block;
+  padding: 0.5rem 1rem;
+  text-decoration: none;
+  border: 1px solid rgba(0,0,0,.125);
+}
```
Comment (sebastian-aranda) [inline love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.module.css:508]: Thanks for refactoring this! I have one last suggestion though:
As you are setting global styles, I recommend to prepend the class of the parent which contains all these elements. One example:
```
.sidePanel :global(.control-label) {
  padding-top: 7px;
  margin-bottom: 0;
  text-align: right;
}
``` 
This way you avoid that other components with same global classes get affected if both of them are present on the same view (the `.sidePanel` is converted to an unique identifier of the component by the CSS modules).

**Review by sebastian-aranda — APPROVED** [review]
Good job, thanks @fdelcampo !

**Pull request merged**